graphsync.go

```diff
@@ -14,6 +15,10 @@ import (
 // RequestID is a unique identifier for a GraphSync request.
 type RequestID int32
 
+func (r RequestID) String() string {
+	return strconv.Itoa(int(r))
```

Comment:
is this preferable to `fmt.Sprintf("%d", r)`?

Reply:
should we have a namespace prefix to prevent potential tag collisions?

Reply:
yea I like that.